certified-connectors/DocuSignDemo/script.csx

```diff
@@ -4156,6 +4156,32 @@ private void RenameSpecificKeys(JObject jObject, Dictionary<string, string> keyM
     return body;
   }
 
+  private JObject SearchListEnvelopesTransformation(JObject body)
+  { 
+      var uriBuilder = new UriBuilder(this.Context.Request.RequestUri);
+      var query = HttpUtility.ParseQueryString(this.Context.Request.RequestUri.Query);
+      uriBuilder.Path = uriBuilder.Path.Replace("/SearchListEnvelopes", "");
+
+      query["include"] = "custom_fields, recipients, documents, folders";
+      query["order"] = "desc";
+      
+      query["status"] = string.IsNullOrEmpty(query.Get("envelopeStatus")) ? 
+        null : query.Get("envelopeStatus");
+      query["folder_ids"] = string.IsNullOrEmpty(query.Get("folder_ids")) ? 
```

Comment:
Is this necessary?

Reply:
I believe you can optionally filter based on status. @jeremiahs-docusign please confirm.

Reply:
yeah envelope status is one of the filter options.

Reply:
If we don't have this line, it `folder_id` would still be included in query as long as user adds that input when using this action correct?

Reply:
I removed it and found that when filtering based on status it doesnt pick up on the query for the output. So its necessary to include.

Reply:
hmm got it